### PR TITLE
Docs: Require 3 approvals to merge PR

### PR DESCRIPTION
**Pull request by rlmead**
[SC-7706](https://app.shortcut.com/aoeu-se/story/7706/update-docs-to-require-3-approvals-on-prs)

### CODE REVIEW

File: docs/source-control/new-repo.md
Comment (jmosley5) [inline docs/source-control/new-repo.md:38]: Thank you @rlmead for doing this! Would it be better to clarify that we need 2 dev and 1 qa approval before merging? Context is [here](https://theartofed.slack.com/archives/C0190RSMPQX/p1625080118213200?thread_ts=1625079465.210400&cid=C0190RSMPQX)

Comment (jmosley5) [reply docs/source-control/new-repo.md:38]: Ah! Per a convo with @rlmead, we found relevant information regarding this [here](https://docs.theartofeducation.io/pages/source-control/code-reviews#code-review-approval), so additional clarification is not necessary.

**Review by jmosley5 — APPROVED** [review]
:shipit:

**Review by ryexley — APPROVED** [review]

**Review by Thomas-Jean — APPROVED** [review]

**Pull request merged**
